frontend/www/js/omegaup/group/edit.ts

```diff
@@ -225,15 +236,11 @@ OmegaUp.on('ready', () => {
             hiddenElement.click();
           },
           'read-csv': (
-            source: group_Identities,
-            fileUpload: HTMLInputElement,
+            source: { identities: types.Identity[] },
+            fileUpload: File,
```

Comment:
```suggestion
            { identities: types.Identity[] },
            file: File,
```

si no se necesita el nombre del parámetro se puede usar [Object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) para los parámetros también! entonces en vez de usar `source.identities`, se puede usar `identities` directamente.

habiendo dicho eso, será posible mandar el `file: File` dentro del objeto? como que me gusta más este estilo donde los eventos mandan un sólo parámetro con todas las propiedades necesarias para el receptor del evento.

Reply:
Listo.